m2cgen/ast.py

```diff
@@ -233,6 +309,15 @@ def __str__(self):
         args = ",".join([str(self.left), str(self.right), self.op.name])
         return "CompExpr(" + args + ")"
 
+    def __eq__(self, other):
+        return (type(other) is CompExpr and
```

Comment:
Can we use `_eq_bin_exprs` here too?

Reply:
Technically yes. But `CompExpr` doesn't inherit `BinExpr`, so I'm a little bit confused about semantics. However, I don't have strong opinion here.

Reply:
Well technically it's still a binary expression, just not a numeric one.

Reply:
Done in f7b7309a31689ecdefd3789dbf6ee065f7a6336d.